src/directives/v-disabled.js

```diff
@@ -2,5 +2,8 @@
  * ORIGINAL SOURCE: src/app/gui/vue/directives/v-disabled.js@v3.6
  */
 export default (el, binding) => {
-  el.classList.toggle('g3w-disabled', binding.value);
+  const className = 'g3w-disabled';
+  if (binding.value) {
+    if (!el.classList.contains(className)) el.classList.add(className)
+  } else el.classList.remove(className);
```

Comment:
I'm quite sure that the solution adopted before wasn't equivalent to the one you propose...  (is this just an oversight?)

**Ref:** [DOMTokenList/toggle#setting_the_force_parameter](https://developer.mozilla.org/en-US/docs/Web/API/DOMTokenList/toggle#setting_the_force_parameter)